eva_submission/eload_brokering.py

```diff
@@ -233,6 +235,43 @@ def _ena_report(self):
     - receipt: {receipt}
 """.format(**report_data))
         return '\n'.join(reports)
+    
+    def _archival_confirmation_text(self):
+        metadata_file = self.eload_cfg.query('submission', 'metadata_spreadsheet')
+        reader = EvaXlsxReader(metadata_file)
+        study_title = reader.project_title
```

Comment:
That works but maybe we should have added it to the config earlier so we don't need to get it back. What do you think ?

Reply:
done